### ISSUE TITLE
Cannot migrate in dashboard console

### ISSUE THREAD

**Issue by cliffoo** [trufflesuite/truffle#5311]
## Steps to Reproduce

- Start dashboard: `truffle dashboard`
- In a truffle project, connect to dashboard console: `truffle console --network dashboard`
- In the dashboard console, run `migrate`
- Fail

## `migrate` in _console_ is broken but 

`truffle migrate --network dashboard` works just fine 🤨

Other things work ok in console too, like `compile` and `SomeContract.new()`

## Which release introduced this?

I wish I knew which _commit_ introduced this. This issue exists on Truffle >= v5.5.20 (and not on < v5.5.20) 

## Actual Results

```
truffle(dashboard)> migrate

Compiling your contracts...
===========================
> Everything is up to date, there is nothing to compile.
> Something went wrong while attempting to connect to the network at http://127.0.0.1:9545/. Check your network configuration.
ProviderError: 
Could not connect to your Ethereum client with the following parameters:
    - host       > 127.0.0.1
    - port       > 9545
    - network_id > *
Please check that your Ethereum client:
    - is running
    - is accepting RPC connections (i.e., "--rpc" or "--http" option is used in geth)
    - is accessible over the network
    - is properly configured in your Truffle configuration file (truffle-config.js)

    at /Users/cliff/.nvm/versions/node/v16.15.0/lib/node_modules/truffle/build/webpack:/packages/provider/wrapper.js:76:1
    at /Users/cliff/.nvm/versions/node/v16.15.0/lib/node_modules/truffle/build/webpack:/packages/provider/wrapper.js:114:1
    at XMLHttpRequest.request.onreadystatechange (/Users/cliff/.nvm/versions/node/v16.15.0/lib/node_modules/truffle/build/webpack:/node_modules/web3-providers-http/lib/index.js:98:1)
    at XMLHttpRequestEventTarget.dispatchEvent (/Users/cliff/.nvm/versions/node/v16.15.0/lib/node_modules/truffle/build/webpack:/node_modules/xhr2-cookies/dist/xml-http-request-event-target.js:34:1)
    at XMLHttpRequest.exports.modules.996763.XMLHttpRequest._setReadyState (/Users/cliff/.nvm/versions/node/v16.15.0/lib/node_modules/truffle/build/webpack:/node_modules/xhr2-cookies/dist/xml-http-request.js:208:1)
    at XMLHttpRequest.exports.modules.996763.XMLHttpRequest._onHttpRequestError (/Users/cliff/.nvm/versions/node/v16.15.0/lib/node_modules/truffle/build/webpack:/node_modules/xhr2-cookies/dist/xml-http-request.js:349:1)
    at ClientRequest.<anonymous> (/Users/cliff/.nvm/versions/node/v16.15.0/lib/node_modules/truffle/build/webpack:/node_modules/xhr2-cookies/dist/xml-http-request.js:252:48)
    at ClientRequest.emit (node:events:527:28)
    at Socket.socketErrorListener (node:_http_client:454:9)
    at Socket.emit (node:events:527:28)
    at emitErrorNT (node:internal/streams/destroy:157:8)
    at emitErrorCloseNT (node:internal/streams/destroy:122:3)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)
```

## Environment

* Operating System: Monterey
* Ethereum client: Ganache v7.3.2
* Truffle version (`truffle version`):
* node version (`node --version`):
* npm version (`npm --version`):

**Comment by sukanyaparashar** [comment]
Hey @cliffoo. Thanks for raising this. This is a known issue for the `truffle console` and `truffle develop` with `dashboard` and public testnets. This PR #5329 is currently up for resolving this.

**Comment by haltman-at** [comment]
This should be fixed in the latest Truffle release (5.5.25), so I'm closing this.  Obviously go ahead and reopen this if there's still a problem!
